app/src/main/java/eu/kanade/presentation/browse/ExtensionsScreen.kt

```diff
@@ -278,16 +281,47 @@ private fun ExtensionItem(
     onClickItemAction: (Extension) -> Unit,
     onClickItemSecondaryAction: (Extension) -> Unit,
     modifier: Modifier = Modifier,
+    context: Context,
 ) {
     val (extension, installStep) = item
+    var showUninstallConfirmation by remember { mutableStateOf(false) }
+
+    if (showUninstallConfirmation) {
+        ExtensionUninstallConfirmation(
+            extensionName = extension.name,
+            onClickConfirm = {
+                onLongClickItem(extension)
+                showUninstallConfirmation = false
+            },
+            onClickDismiss = {
+                showUninstallConfirmation = false
+            },
+            onDismissRequest = {
+                showUninstallConfirmation = false
+            },
+        )
+    }
+
     BaseBrowseItem(
         modifier = modifier
             .combinedClickable(
                 onClick = { onClickItem(extension) },
-                onLongClick = { onLongClickItem(extension) },
+                onLongClick = {
+                    if (context.isPackageInstalled(extension.pkgName)) {
+                        onLongClickItem(extension)
+                    } else {
+                        showUninstallConfirmation = true
+                    }
+                },
             ),
         onClickItem = { onClickItem(extension) },
-        onLongClickItem = { onLongClickItem(extension) },
+        onLongClickItem = {
+            if (context.isPackageInstalled(extension.pkgName)) {
+                onLongClickItem(extension)
+            } else {
+                showUninstallConfirmation = true
+            }
```

Comment:
Same as above. May also want to make a lambda function like `internalOnLongClickItem` so it's shared.